lib/linguist/languages.yml

```diff
@@ -7583,6 +7583,8 @@ Vim Script:
   - vim
   - viml
   - nvim
+  - vim script
```

Comment:
This is not needed as this is just the lowercase version of the current language.

```suggestion
```